pom.xml

```diff
@@ -286,7 +287,13 @@
                         <dependency>
                             <groupId>org.junit.vintage</groupId>
                             <artifactId>junit-vintage-engine</artifactId>
-                            <version>5.7.0</version>
+                            <version>${junit.version}</version>
+                        </dependency>
+                        <!-- JUnit5 Platform Engine for Junit 5 -->
+                        <dependency>
+                            <groupId>org.junit.jupiter</groupId>
+                            <artifactId>junit-jupiter-engine</artifactId>
+                            <version>${junit.version}</version>
```

Comment:
By adding this here additionally surefire will now find junit3, junit4, junit5 and kotlin tests.
Ideally we can remove this extra config in the end.